src/components/radio-button-group/radio-button-group.e2e.ts

```diff
@@ -418,6 +418,12 @@ describe("calcite-radio-button-group", () => {
       </calcite-radio-button-group>
     `);
 
+    const getSelectedItemValue = async (): Promise<string> => {
```

Comment:
Nice! This was added in the Stepper PR as well, maybe a good candidate for helper util?